src/styles/_utils/_media.scss

```diff
@@ -0,0 +1,17 @@
+@media screen and (min-width: 767px) and (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media screen and (min-width: 488px) and (max-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media screen and (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
```

Comment:
Let'v move repeated values to variables.

Reply:
You mean cards width move to vars?